dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/CaseNode.java

```diff
@@ -24,30 +23,34 @@
   /** The switch expression. */
   protected final Node switchExpr;
   /** The case expression to match the switch expression against. */
-  protected final Node caseExpr;
+  protected final List<Node> caseExprs;
 
   /**
    * Create a new CaseNode.
    *
    * @param tree the tree for this node
    * @param switchExpr the switch expression
-   * @param caseExpr the case expression to match the switch expression against
+   * @param caseExprs the case expression(s) to match the switch expression against
    * @param types a factory of utility methods for operating on types
    */
-  public CaseNode(CaseTree tree, Node switchExpr, Node caseExpr, Types types) {
+  public CaseNode(CaseTree tree, Node switchExpr, List<Node> caseExprs, Types types) {
     super(types.getNoType(TypeKind.NONE));
     assert tree.getKind() == Kind.CASE;
     this.tree = tree;
     this.switchExpr = switchExpr;
-    this.caseExpr = caseExpr;
+    this.caseExprs = caseExprs;
   }
 
   public Node getSwitchOperand() {
     return switchExpr;
   }
 
-  public Node getCaseOperand() {
-    return caseExpr;
+  /**
+   * Gets the nodes corresponding to the case expressions. There can be multiple expressions since
+   * Java 12.
```

Comment:
The formatting rules will fail unless you also add a `@return` tag to this Javadoc comment.